ONSdigital/eq-survey-runner
# Add test survey

### Changes

Adds a large test survey as code (Survey itself is a work in progress)
### How to test

1) checkout the branch
2) run the app and visit the dev console
3) choose 99_test_survey from the schema dropdown
4) play with the test_survey schema
5) make changes to the code in app/test_survey
6) run `python export_test_survey.py`
7) Run the survey again to check your changes
### Who can test

Anyone except @weapdiv-david
